api-contract/src/test/java/uk/gov/hmcts/payment/api/contract/CreditAccountPaymentRequestTest.java

```diff
@@ -96,6 +96,20 @@ public void testValidateSiteIdForFpl() {
         );
     }
 
+    @Test
+    public void testValidateSiteIdForAdoption() {
```

Comment:
I think CreditAccountPaymentRequestTest.java  is for PBA payments not card payments

Reply:
its done Yashar .. moved to card payment.. thanks